src/Sulu/Bundle/ContactBundle/Contact/ContactManagerInterface.php

```diff
@@ -40,21 +43,21 @@ public function removeAddressRelation($entity, $address);
     /**
      * Returns a collection of relations to get addresses.
      *
-     * @param string $entity
+     * @param ContactInterface $entity
      */
     public function getAddressRelations($entity);
 
     /**
      * sets the first element to main, if none is set.
      *
-     * @param array $arrayCollection
+     * @param ArrayCollection $arrayCollection
      */
     public function setMainForCollection($arrayCollection);
 
     /**
      * unsets main of all elements of an ArrayCollection | PersistanceCollection.
      *
-     * @param array $arrayCollection
+     * @param ArrayCollection $arrayCollection
```

Comment:
As the comment above mention this can be both a `ArrayCollection` (used by doctrine when a new entity is created) and a `PersistentCollection` (used by doctrine when an exist entity is edited).

They have a common class called `Collection` so I would use that in the typehint.